scripts/new_project.kts

```diff
@@ -232,8 +242,9 @@ object NewProject {
         sourceFile.writeText(sourceText)
     }
 
-    private fun showMessage(message: String) {
+    private fun showMessage(message: String, exitAfterMessage: Boolean = false) {
         println("\n${message}\n")
+        if (exitAfterMessage) System.exit(0)
```

Comment:
Should we re-factor `showMessage()` and `showErrorMessage()` in to one function ? 🙇